### PR TITLE
feat: Add multi-arch support in sbom-json-check task

### PR DESCRIPTION
**Pull request by yashvardhannanavati**
Refers to CVP-4048. With multi-arch builds, the sbom-json-check task is able to identify and consume an Image Index as input and check all Image Manifests in the Image Index.

# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

### CODE REVIEW

**Comment by yashvardhannanavati** [comment]
@dirgim @MartinBasti @jsztuka @Josh-Everett Can I please request a review here?

File: task/sbom-json-check/0.1/sbom-json-check.yaml
Comment (MartinBasti) [inline task/sbom-json-check/0.1/sbom-json-check.yaml:23]: needs newer image to work

Comment (yashvardhannanavati) [reply task/sbom-json-check/0.1/sbom-json-check.yaml]: yes, was waiting for a new version to release. Thanks for merging that PR. Updated the image here

File: task/sbom-json-check/0.1/sbom-json-check.yaml
Comment (dirgim) [inline task/sbom-json-check/0.1/sbom-json-check.yaml:53]: nit: this comment isn't necessary anymore since `get_image_manifests` now fetches the image even if it isn't a manifest list.

File: task/sbom-json-check/0.1/sbom-json-check.yaml
Comment (dirgim) [inline task/sbom-json-check/0.1/sbom-json-check.yaml:49]: nit: would `echo -n` be simpler here to avoid the newline?

**Comment by jsztuka** [comment]
Changes looks good to me, once all the checks pass, I will provide approval.

File: task/sbom-json-check/0.1/sbom-json-check.yaml
Comment (yashvardhannanavati) [reply task/sbom-json-check/0.1/sbom-json-check.yaml:49]: yes, absolutely!

**Review by jsztuka — APPROVED** [review]
LGTM

**Review by dirgim — APPROVED** [review]
LGTM

**Pull request merged**
